docs/topics/gradle/gradle-backend-tutorial.md

```diff
@@ -0,0 +1,107 @@
+[//]: # (title: Get started with Gradle and Kotlin/JVM)
+
+This tutorial demonstrates how to use IntelliJ IDEA and Gradle for creating a console application.
+
+To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
+
+## Create a project
+
+1. In IntelliJ IDEA, select **File** | **New** | **Project**.
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location if necessary.
+
+   > Select the **Create Git repository** checkbox to place the new project under version control. You will be able to do
+   > it later at any time.
+   >
+   {type="tip"}
+
+4. From the **Language** list, select **Kotlin**.
+
+   ![Create a console application](jvm-new-gradle-project.png){width=700}
+
+5. Select the **Gradle** build system. Choose the Kotlin language for the build script.
+6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
+   your project.
+    * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
+      JDK home directory.
+    * If you don't have the necessary JDK on your computer, select **Download JDK**.
+
+7. Enable the **Add sample code** option to create a file with a sample `"Hello World!"` application.
+8. Click **Create**.
+
+## Understand the build script
+
+Open the `build.gradle.kts` file: it is the Gradle Kotlin build script, which contains a list of the dependencies required for the application.
+
+Here is the full script with the explanation of all parts and dependencies:
+
+```kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile // For `KotlinCompile` task below
```

Comment:
I find these comments a bit "too much" maybe.
<img width="512" alt="Screenshot 2022-10-18 at 17 18 11" src="https://user-images.githubusercontent.com/78360457/196472270-d06204e6-36f1-40aa-b9bb-b1cad6424204.png">

I like how it's done in the [Gradle docs](https://docs.gradle.org/current/samples/sample_building_kotlin_applications.html#review_the_project_files):
<img width="580" alt="Screenshot 2022-10-18 at 17 19 21" src="https://user-images.githubusercontent.com/78360457/196472581-28b1fa9f-4ffb-4031-bee5-b8d3f1351e2e.png">

I haven't explored yet if Webhelp allows us to do the same. Will research in the nearest time. If you know, please, ping me.

Reply:
We can use emojis if we want to have something similar:

<img width="645" alt="Screenshot 2022-10-21 at 20 07 44" src="https://user-images.githubusercontent.com/78360457/197261531-fdf579ba-6add-453d-a2cc-d98dfbd8c389.png">

Reply:
But you can leave these numbers if you want to provide some more information on this line of code, to keep the comment as short and concise as possible